easybuild/easyconfigs/p/PPanGGOLiN/PPanGGOLiN-1.1.136-foss-2020b.eb

```diff
@@ -0,0 +1,74 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Denis Kristak
+easyblock = 'PythonPackage'
+
+name = 'PPanGGOLiN'
+version = '1.1.136'
+
+homepage = 'https://github.com/labgem/PPanGGOLiN'
+description = """
+PPanGGOLiN is a software suite used to create and manipulate prokaryotic
+pangenomes from a set of either genomic DNA sequences or provided genome annotations. It is designed
+to scale up to tens of thousands of genomes. It has the specificity to partition the pangenome using
+a statistical approach rather than using fixed thresholds which gives it the ability to work with low-quality
+data such as Metagenomic Assembled Genomes (MAGs) or Single-cell Amplified Genomes (SAGs) thus taking advantage
+of large scale environmental studies and letting users study the pangenome of uncultivable species.
+"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+source_urls = ['https://github.com/labgem/%(name)s/archive/refs/tags/']
+sources = ['%(version)s.tar.gz']
+patches = ['PPanGGOLiN-1.1.136_new_compiler_compatibility.patch']
+checksums = [
+    '41f814f0b9331a8f0e04f09c698cef6200429990ce726339b5bb320bb7943a70',  # 1.1.136.tar.gz
+    # PPanGGOLiN-1.1.136_new_compiler_compatibility.patch
+    'ae4dadf2f7469e4c3d56b10236aa7812bfe6f5ed231676364f7c3f0d42e8facf',
+]
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('tqdm', '4.56.2'),
+    ('PyTables', '3.6.1'),
+    ('prodigal', '2.6.3'),
+    ('ARAGORN', '1.2.41'),
+    ('Infernal', '1.1.4'),
+    ('MMseqs2', '13-45111'),
+    ('networkx', '2.5'),
+    ('SciPy-bundle', '2020.11'),
+    ('plotly.py', '4.14.3'),
+    ('gmpy2', '2.1.0b5'),
+    ('genoPlotR', '0.8.11'),
+    ('MAFFT', '7.475', '-with-extensions'),
+    ('rpy2', '3.4.5'),
+]
+
+configopts = '-DBLAS_LIBRARIES=$EBROOTOPENBLAS/lib/libopenblas.%s ' % SHLIB_EXT
+configopts += '-DLAPACK_LIBRARIES=$EBROOTOPENBLAS/lib/libopenblas.%s ' % SHLIB_EXT
+
+use_pip = True
+sanity_pip_check = True
+download_dep_fail = True
+
+exts_default_options = {
+    'source_urls': [PYPI_SOURCE],
+    'use_pip': True,
+    'download_dep_fail': True,
+
+}
+exts_defaultclass = 'PythonPackage'
```

Comment:
Using `PythonPackage` to install extensions in an easyconfig using `PythonPackage` as easyblock doesn't make much sense, you should be able to clean this up a bit by using `PythonBundle` as an easyblock for this easyconfig instead.

Reply:
done